cmd/root.go

```diff
@@ -34,10 +36,12 @@ func init() {
 
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", fmt.Sprintf("configuration file, default %s", cliConfigPath()))
 	rootCmd.PersistentFlags().StringVar(&account, "account", "", "the account used, 'default' if none given")
+	rootCmd.PersistentFlags().BoolVarP(&jsonFlag, "json", "j", false, "Returns a table as json.")
```

Comment:
Maybe better "Print results as JSON"" or "Print output as JSON". Also note: flags and help texts don't end with a trailing dot.

Reply:
fixed

Reply:
Ehm, no. I meant the other way around. Sorry my wording wasn't very clear.

Flags and help texts should _not_ end with a trailing dot.

So, no dot.